ci-operator/step-registry/network/calico/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+- smarterclayton
+- wking
+- vrutkovs
+- abhinavdahiya
+- deads2k
```

Comment:
Do we have a Calico maintainer we can add here?

Reply:
We attempted to do that in https://github.com/openshift/release/pull/17380, but `openshift-bot` was not happy - owners must be in openshift org.